templates/alumniprofile/previous_edu_form.html

```diff
@@ -9,10 +9,15 @@ <h4>Add Education</h4>
       <label for="degree" class="col-form-label">
         Degree
       </label>
-      <input type="text" placeholder="M.Tech" name="degree" maxlength="200" class="form-control textinput" id="degree"
-      required>
+      <input type="text" placeholder="Enter Your Degree" name="degree" maxlength="200" class="form-control textinput" id="degree_input" style="display: none;">
+    <select name="degree" id="degree_select" class="custom-select">
+      <option value="" selected disabled hidden>Select Degree</option>
+      {% for d in DEGREE %}
+        <option value="{{d}}">{{d}}</option>
+      {% endfor %}
```

Comment:
You could have just added an "Other" option here, choosing which will make the text field visible.